react/features/base/lib-jitsi-meet/actionTypes.js

```diff
@@ -68,3 +68,13 @@ export const SET_CONFIG = Symbol('SET_CONFIG');
  * }
  */
 export const SET_WEBRTC_READY = Symbol('SET_WEBRTC_READY');
+
+/**
+ * The type of Redux action which signals that config store in the store should
+ * be merged with one passed in payload.
+ *
+ * {
+ *      type: UPDATE_CONFIG,
+ *      config: Object
+ */
+export const UPDATE_CONFIG = Symbol('UPDATE_CONFIG');
```

Comment:
I don't think we need this if we put the token in the right place, but if we do, make it a separate commit.